manifests/06-operator.yaml

```diff
@@ -29,6 +29,10 @@ spec:
         operator: "Exists"
         effect: "NoExecute"
         tolerationSeconds: 120 # Evict pods within 2 mins.
+      volumes:
+      - name: samples-operator-tls
+        secret:
+          secretName: samples-operator-tls
```

Comment:
do you have logic to handle restarting your metrics server/reloading the tls key when it is rotated?  i didn't see it on my first pass through this PR.

Reply:
I do not .... most likely the image-registry operator already had this and @ricardomaraschini 's PR did not have to add it .... I'll look to see what the existing img reg op is doing and general and adjust

Reply:
I don't think Image Registry Operator nicely handles this. Good catch, i will take a look.

Reply:
will leave this open to handle rotation

also will note here that I was seeing some failed mounts of the secret when I had "non-ideal" manifest ordering .... have addressed that at least with some file renames

Reply:
OK I have an initial pass at restarting the metrics server, though I'm somewhat worried about goroutine sprawl .... my initial investigations did not uncover an existing pattern to copy.

Reply:
.... maybe create a separate signal handler / stop channel for the http server vs. what is used for the controller workqueues, etc.....